core/arch/arm/plat-rockchip/main.c

```diff
@@ -24,7 +24,11 @@ register_phys_mem_pgdir(MEM_AREA_IO_SEC, GIC_BASE, GIC_SIZE);
 
 void boot_primary_init_intc(void)
 {
+	#ifdef PLATFORM_FLAVOR_rk3566
```

Comment:
Please check for `GICR_BASE` or if `CFG_ARM_GICV3` is enabled.
You don't need to indent preprocessor directives.